### PR TITLE
increase performance

### PR DESCRIPTION
**Pull request by Ioniq3**
I only note the improvement in a raspberry pi 0 w

### CODE REVIEW

File: aa_wireless_dongle/board/raspberrypi/config.txt [hunk 1]
```diff
@@ -10,6 +10,7 @@ disable_poe_fan=1
 # Disable overscan assuming the display supports displaying the full resolution
 # If the text shown on the screen disappears off the edge, comment this out
 disable_overscan=1
+hdmi_blanking=1
```
Comment (nisargjhaveri) [inline aa_wireless_dongle/board/raspberrypi/config.txt:13]: Can we add a short comment on why we're doing this?

Comment (Ioniq3) [reply aa_wireless_dongle/board/raspberrypi/config.txt:13]: Of course, in a buildroot you cannot shutdown the hdmi output by "tvservice -o" so the best way to disable hdmi port is in config.txt.
Disabling hdmi you reduce the power the board needs and the i/o of the processor, so weak board lke rpi 0w can do the work with best performance

Comment (gamelaster) [reply aa_wireless_dongle/board/raspberrypi/config.txt:13]: @Ioniq3 I think @nisargjhaveri meant to add the comment into file.

Comment (Ioniq3) [reply aa_wireless_dongle/board/raspberrypi/config.txt:14]: Great, added

**Comment by nisargjhaveri** [comment]
Merged. Thanks for this! :)

**Pull request merged**